src/app/storage/utils.ts

```diff
@@ -0,0 +1,72 @@
+import axios, { Axios, AxiosError, AxiosResponse, Method } from 'axios';
+
+import { INITIAL_STORAGE } from './constants';
+import { OverallStorageObject, StorageResponse } from './types';
+import { PatchInstruction } from './types/json-patch';
+
+const token = `
+eyJraWQiOiIyZDM3OTRlOC1iMmJhLTMxYWItYTVmNC0xMjExNDZiZGUxZmMiLCJhbGciOiJSUzI1NiJ9.eyJ0b2tlbktpbmQiOiJiZWFyZXIiLCJqdGkiOiJkMmQzMzUxMC03MzgyLTRhZGMtOWExZi01ZDE1YmVlZmU5NmYiLCJhdWQiOlsiaHR0cDovL2xvY2FsaG9zdDozMDAwIiwiaHR0cHM6Ly93YWxsZXQuc3RhZ2luZy5kbmFzdGFjay5jb20iXSwiYXpwIjoib2ljci1jbGllbnQiLCJpYXQiOjE2MzY2NjEwMzEsImV4cCI6MTYzNjc0NzQzMSwic3ViIjoiNzQyMTY1ZTUtNWZkMy00ZmEwLTkwYmUtODUyZGFiZjY3ODlkIiwiaXNzIjoiaHR0cHM6Ly93YWxsZXQuc3RhZ2luZy5kbmFzdGFjay5jb20iLCJzY29wZSI6InVzZXItc3RvcmFnZSBvcGVuaWQgb2ZmbGluZV9hY2Nlc3MiLCJhY3Rpb25zIjp7fX0.OLjss1PsvF4SW2_KbF0JmGzRlCWexq42Sej-MT5oM1CsA2ZYaP0F0z0nTQSYLEGoxnUcPkWokkT2vW4ynIm96aeu9TOb-SB4iD1VINT6LD3hLGdZKpcsipb1e3sFc0y-waZPt4A-MDizcWPfMZVyjqX6N4EIAuNyqSGwuBswI73Cm_4OnEUuoX7_JamPsE9_fP3LuZSg26bvR9GBouW6QnpqZPEevzFmCorVx34ValNW046-KalhtIdo6dmaJIZdPKcYvrzBK13vxU5RjxeJfqtmu2gymWn_f-nayROijRSQ9c2LqvczuufnnTuM5U9aL3qT3ZLURhqUWIpprmf_6Q
```

Comment:
amending this in an upcoming commit, along with the UI integration, now that `/auth` has been merged to develop.